### PR TITLE
feat: done Cards

### PR DESCRIPTION
**Pull request by tushargoyal812**
made responsive cards

https://user-images.githubusercontent.com/69681896/153564496-cef375b8-5b47-49f3-a7e0-2b1cd2db77b6.mp4

### CODE REVIEW

File: components/cards/card.css [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+
+.card {
+    width: 18rem;
+    background-color: var(--white-color);
+    border-radius: 5px;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    position: relative;
+    margin: auto;
+    box-shadow: 0 0 2px 1px #c5c5c5;
```
Comment (arjunbharti) [inline components/cards/card.css:28]: You can use color variable here

File: components/cards/card.css [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+.product-img {
+
+    width: 100%;
+    border-radius: 5px;
+}
+
+.card-container {
+    width: 28rem;
+    border: 2px solid var(--gray-color);
+    margin: auto;
```
Comment (mohitdhatrak) [inline components/cards/card.css:14]: dont name variables with color names. name it generic. Like --border-color etc.

File: components/cards/card.css [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+    border-radius: 5px;
+    background-color: var(--bg-color);
+    padding: 1rem 0rem;
+}
+
+
+.card {
+    width: 18rem;
+    background-color: var(--white-color);
```
Comment (mohitdhatrak) [inline components/cards/card.css:23]: same here

File: components/cards/card.html [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+                            </div>
+                        </main>
+                        <div class="btn-section">
+                            <button class="cart-button">
+                                Add to Cart
+                            </button>
+                        </div>
+                    </div>
+                </div>
+
```
Comment (arjunbharti) [inline components/cards/card.html:149]: Avoid extra spacing

**Review by arjunbharti — COMMENTED** [review]
Added few comments, have a looks. Everything else is fine.

File: components/cards/card.css [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+.card-main-section {
+    padding: 0rem 0.5rem;
+}
+
+.badge-section {
+    padding: 0.5rem;
+}
+
+.para {
+    color: var(--gray-color);
```
Comment (mohitdhatrak) [inline components/cards/card.css:41]: Use better class name here like product-description etc.

File: components/cards/card.css [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+}
+
+
+.card-header {
+    padding: 13px;
+}
+
+.headerCardStyle {
+    margin-top: 10px;
+}
```
Comment (mohitdhatrak) [inline components/cards/card.css:180]: use consistent naming convention for class, use hyphen like everywhere else. Why camelcase

File: components/cards/card.html [hunk 7]
```diff
@@ -0,0 +1,292 @@
...
+                        class="fab fa-github"></i></a>
+                <a href="https://twitter.com/tushargoyal29" class="nav-icons anchor-style"><i
+                        class="fab fa-twitter-square"></i></a>
+                <a href="https://www.linkedin.com/in/tushargoyal29/" class="nav-icons anchor-style"><i
+                        class="fab fa-linkedin"></i></a>
+            </div>
+        </nav>
+    </header>
+    <div class="main-with-sidebar">
+        <aside class="sidebar">
```
Comment (mohitdhatrak) [inline components/cards/card.html:32]: use semantic element like main, section instead of div, if possible

File: components/cards/card.html [hunk 8]
```diff
@@ -0,0 +1,292 @@
...
+                style="width: 100%; height: 562px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (mohitdhatrak) [inline components/cards/card.html:115]: avoid inline css if possible

File: components/cards/card.html [hunk 9]
```diff
@@ -0,0 +1,292 @@
...
+                        Out Of Stock
+                    </span>
+                    <div class="card-main-section">
+                        <img src="/images/IMG_20220205_233718.png" class="product-img" alt="">
+                        <main class="middle">
+                            <p class="para">Apple</p>
+                            <div class="small-1">iPhone 13 (256GB) - Midnight</div>
+                            <div class="small-2">₹78900
+                                <span class="real-price">MRP: ₹79900(1% OFF)</span>
+                            </div>
```
Comment (mohitdhatrak) [inline components/cards/card.html:178]: if there is only 1 element inside div, span, then i guess div container is not needed maybe

**Review by mohitdhatrak — COMMENTED** [review]
Few suggestions from my side, overall looks good!

**Pull request merged**